pages/jobs/[id]/index.jsx

```diff
@@ -15,26 +15,22 @@ function BrowseJobPage({ job }) {
 
   return (
     <>
-      <Head>
-        <title>
-          {job?.title} | {job?.company} | Denver Devs
-        </title>
-        <meta
-          name="description"
-          content={`Looking for a job in Denver? ${job?.company} is hiring! Check out their job posting for a ${job?.title} on Denver Devs.`}
-        />
-        <meta
-          property="og:title"
-          content={`${job?.title} | ${job?.company} | Denver Devs`}
-        />
-        <meta
-          property="og:description"
-          content={`Looking for a job in Denver? ${job?.company} is hiring! Check out their job posting for a ${job?.title} on Denver Devs.`}
-        />
-        <meta property="og:image" content={job?.image} />
-        <meta property="og:url" content={`https://denverdevs.org/jobs/${id}`} />
-        <meta property="og:type" content="website" />
-      </Head>
+      <NextSeo
+        title={`Jobs | ${job?.title} | ${job?.company}`}
+        description={`Looking for a job in Denver? ${job?.company} is hiring! Check out their job posting for a ${job?.title} on Denver Devs.`}
+        openGraph={{
+          url: `https://denverdevs.org/jobs/${id}`,
+          images: [
+            {
+              url: job?.public_logo_url,
```

Comment:
Not an action item, just a note for me to remember but I wonder if we could tie this in with #142 if that gets set up, and generate images with the logo -- that'd be neat!